### DISCUSSION TITLE
195

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#170] [General]
# 潮流周刊第195期 - BELLOCO

封面图拍摄于杭州嘉里中心外面的韩式料理餐厅 BELLOCO，依稀记得之前很难排队，最近消费降级好了不少，环境服务挺好的，值得一去。

https://weekly.tw93.fun/posts/195-BELLOCO

**Comment by xxholly32** [comment]
A ship is always safe at shore, but that is not what it’s built for. 这句是 托尼 github 主页的那句，https://github.com/antfu

**Reply by tw93** [reply]
原来如此！

**Comment by laloe74** [comment]
那句话是印在哪本书上的？中文字体好漂亮。

**Reply by tw93** [reply]
随便看到的，不过这句话，你可以Google下，有地方说

**Comment by zianzy** [comment]
今早洗澡过于用力，用这个无印良品刷子搓背，直接把刷子撅成了三截。。

**Reply by tw93** [reply]
哈哈，这里力气应该超级大了

**Comment by wtuyp** [comment]
“造船的目。”，目后面少了一个字“的”

**Reply by tw93** [reply]
谢谢指出，已经修正

**Comment by gzchneg** [comment]
学习了！

**Comment by Hakunassan** [comment]
但这并不是造船的的目。   还是修正错了？hhhh ----“但这并不是造船的目的。”

**Reply by tw93** [reply]
哈哈哈哈哈 好 终于修改好了。

**Comment by 3kaiu** [comment]
搓澡泥无印有吗，没看到唉

**Reply by tw93** [reply]
有的，jd 找到店铺查查，有三种味道

**Comment by ChialinSung** [comment]
muji刷子软毛的好用还是硬毛的好用呀？

**Reply by tw93** [reply]
假如是男，同时之前喜欢用搓澡的，就买普通的，假如是第一次用，可以软毛的

**Comment by isnotry** [comment]
"留在港口的小船最安全，亲爱的，但这不是造船的目的。"
出自弗雷德里克·巴克曼《焦虑的人》
